mw-webapp/src/component/title/Title.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {Heading} from "@radix-ui/themes";
+import styles from "./Title.module.scss";
+
+export enum HeadingLevel {
+  H1 = "h1",
+  H2 = "h2",
+  H3 = "h3",
+  H4 = "h4",
+  H5 = "h5",
+  H6 = "h6",
+}
+
+interface TitleProps {
+  /**
+   * Heading level
+   */
+  level: HeadingLevel;
+  /**
+   * CSS class name for styling the component.
```

Comment:
good comments in general but we still could improve them a little! See, we don't have another className not for styling here, so looks like in this context this word means nothing. That means we could get read of it.